.circleci/config.yml

```diff
@@ -43,31 +55,45 @@ jobs:
       BLOCKCHAIN_PATH: ./blockchain/    
     steps:
       - checkout
-      - run: 
-          name: Install scripts dependencies
-          command: pip3 install -r scripts/requirements.txt
+      - run:
+          name: Create virtual environment
+          command: |
+            python3 -m venv venv/
+            source venv/bin/activate
+            pip3 install -r Connector/requirements.txt
+            pip3 install -r scripts/requirements.txt
       - run: 
           name: Check Lint
-          command: flake8 --statistics
+          command: |
+            source venv/bin/activate
+            flake8 --statistics
       - run: 
-          name: Deploy Development BTC Connector
+          name: Deploy BTC development nodes
           command: |
             docker-compose -f Connector/tests/docker-compose/btc.yml -p btc_development_api up --build -d
+      - run:
+          name: Add electrum and bitcoincore to /etc/hosts
+          command: |
+            echo "127.0.0.1 electrum" | sudo tee -a /etc/hosts
+            echo "127.0.0.1 bitcoincore" | sudo tee -a /etc/hosts
+            cat /etc/hosts
```

Comment:
Same question

Reply:
Line 77 and 78 are mandatory, line 79 just for logging